modules/nf-core/bwa/index/meta.yml

```diff
@@ -11,7 +11,7 @@ tools:
         BWA is a software package for mapping DNA sequences against
         a large reference genome, such as the human genome.
       homepage: http://bio-bwa.sourceforge.net/
-      documentation: http://www.htslib.org/doc/samtools.html
+      documentation: http://bio-bwa.sourceforge.net/
```

Comment:
```suggestion
      documentation: https://bio-bwa.sourceforge.net/bwa.shtml
```

Reply:
Yeah that's fine. But then I'll update all of the others as well so it's all the same.